Tests/Tests.csproj

```diff
@@ -18,6 +18,8 @@
 
   <ItemGroup>
     <PackageReference Include="EasyNetQ.Management.Client" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics.Testing" />
+    <PackageReference Include="Microsoft.Extensions.Diagnostics" Condition="'$(TargetFramework)'=='net8.0'" />
```

Comment:
Why do we need this condition? `Condition="'$(TargetFramework)'=='net8.0'" ` ?

Reply:
Because their only available on version 8 (corrrelated to the IMeterFactory pattern that allow better testability of Metrics) => https://www.nuget.org/packages/Microsoft.Extensions.Diagnostics#versions-body-tab 
So this dependecy will clash with SYstem Diagnostic one define in lower framework version

Reply:
@aygalinc, thank you. Can you give me the grants to commit to this repo? I'd need to format the code and run all the tests.

Reply:
I think you have the right to edit the PR and you have previously done this by formatting the code. I will check to format it on my side

Reply:
@Gsantomaggio i have run the make format command it should be good now